### PR TITLE
Add github actions to renovate

### PR DESCRIPTION
**Pull request by michalchecinski**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

Add github actions updates to renovate.json.

## Code changes

* **renovate.json:** Add github actions.

## Before you submit
- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by withinfocus — CHANGES_REQUESTED** [review]

File: .github/renovate.json [hunk 1]
```diff
@@ -17,6 +17,11 @@
         "groupName": "Nuget updates",
         "groupSlug": "nuget",
         "separateMajorMinor": false
+      },
+      {
+        "groupName": "gh minor",
+        "matchManagers": ["github-actions"],
```
[the remark this answers is not in the document]
Comment (michalchecinski) [reply .github/renovate.json:23]: Fixed.

[the remark this answers is not in the document]
Comment (withinfocus) [reply .github/renovate.json:23]: What I meant is to copy the whole file from the template and replace this. Some of the config above like the schedule, dependency dashboard, etc. are incorrect and your change won't work as we expect.

**Review by withinfocus — CHANGES_REQUESTED** [review]

**Comment by michalchecinski** [comment]
Thanks for explaining @withinfocus. Hope it's all good now :)

File: .github/renovate.json [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
+    ":combinePatchMinorReleases",
+    ":dependencyDashboard",
+    ":maintainLockFilesWeekly",
+    ":pinAllExceptPeerDependencies",
+    ":prConcurrentLimit10",
+    ":rebaseStalePrs",
+    "schedule:weekends",
+    ":separateMajorReleases"
+  ],
+  "enabledManagers": ["cargo", "github-actions", "npm", "nuget"],
```
Comment (withinfocus) [inline .github/renovate.json:14]: 💭 I am guessing not all of these are used, but this should be fine to leave as-is.

**Review by withinfocus — CHANGES_REQUESTED** [review]
One small thing!

**Pull request merged**